src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -246,8 +250,38 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
     session.executeAsync(stmt).map(_ => ())
   }
 
-  def partitionNr(sequenceNr: Long): Long =
-    (sequenceNr - 1L) / targetPartitionSize
+  private def readHighestSequenceNr(persistenceId: String, fromSequenceNr: Long): Long = {
```

Comment:
These methods were moved from CassandraRecovery and are now private to CassandraJournal. They are not needed in recovery anymore, because it uses the async versions.